modules/swagger-codegen/src/main/resources/php/ObjectSerializer.mustache

```diff
@@ -241,6 +241,8 @@ class ObjectSerializer
                 $values[] = $this->deserialize($value, $subClass);
             }
             $deserialized = $values;
+        } elseif ($class === 'ByteArray') { // byte array
+            $deserialized = unpack('C*', (string)$data);
```

Comment:
I realize the mapped type is a technically a byte array but this seems more natural to return as a string. Even printing this data is complicated as it's an array of integers and PHP's `pack` cannot be called with an `Array`.

```php
$string = "";
foreach($deserialized as $int) {
  $string = $string . pack("C", $int);
}
```

Compare with string returned:

```php
echo $deserialized;
```

File operations also don't support this unpacked format either.